cmd/krew/cmd/search.go

```diff
@@ -54,10 +54,14 @@ Examples:
 			pluginMap[p.Name] = p
 		}
 
-		installed, err := installation.ListInstalledPlugins(paths.InstallReceiptsPath())
+		receipts, err := installation.GetInstalledPluginReceipts(paths.InstallReceiptsPath())
 		if err != nil {
 			return errors.Wrap(err, "failed to load installed plugins")
 		}
+		installed := make(map[string]string)
+		for _, receipt := range receipts {
+			installed[receipt.Name] = receipt.Spec.Version
```

Comment:
this sounds wrong. what if two plugins with identical names from different indexes exist? it'll show both as installed.

Reply:
This is here to preserve the existing functionality. I'm not trying to update search to work with custom indexes in this PR.

Reply:
Yeah makes sense. That said do you mind adding a TODO so we don't forget these things? The same works below, too.